### PR TITLE
FiltersBar from Goals

### LINKED ISSUE #1153: FiltersBar from Goals

### PR DESCRIPTION
**Pull request by IgorGoryany**
## PR includes

- [x] Feature

## Related issues

Resolve #1153

## QA Instructions, Screenshots, Recordings

Снимок экрана 2024-08-28 в 17 50 51

### CODE REVIEW

**Review by IgorGoryany — COMMENTED** [review]
/canary

File: src/harmony/FiltersBar/FiltersBar.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { StoryFn, Meta } from '@storybook/react';
+
+import { Button } from '../Button/Button';
+import { Separator } from '../Separator/Separator';
```
Comment (asabotovich) [inline src/harmony/FiltersBar/FiltersBar.stories.tsx:5]: fix warn

**Review by asabotovich — APPROVED** [review]

**Review by tsumo — APPROVED** [review]

**Pull request merged**